src/utils/LandCategories.js

```diff
@@ -1,34 +1,4 @@
 export default {
-  CREATURE: [
```

Comment:
Can we make this a json file and put it in res with the casting probability table?